website/server/models/group.js

```diff
@@ -35,7 +35,7 @@ import stripePayments from '../libs/stripePayments';
 const questScrolls = shared.content.quests;
 const Schema = mongoose.Schema;
 
-export const INVITES_LIMIT = 100;
+export const INVITES_LIMIT = 100; // must not be greater than MAX_EMAIL_INVITES_BY_USER
```

Comment:
There's more details in the commit message and in slack but if you haven't yet reached MAX_EMAIL_INVITES_BY_USER, you can send as many invitations as you want in a single API call, and so if INVITES_LIMIT is greater than MAX_EMAIL_INVITES_BY_USER, it's a once-off way to send more than MAX_EMAIL_INVITES_BY_USER. It's a minor exploit though so not worth fixing (or at least not worth fixing now when we have to get this deployed quickly).